src/TestCS.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Leverage\Toolchain;
+
+use http\Env\Request;
+
+class TestCS
+{
+    public function foo(): void
+    {
+        $value = '1';
+        /** @var Request $payload */
+        $payload = new class ($value) extends Request
+        {
+            public function __construct(
+                public readonly string $value,
+                public readonly ?string $notSet = null,
+            ) {
+            }
+        };
+    }
+
+    public function bar(): void
+    {
+        $this->request = new class (
+            payload: $this->payload,
+        ) extends Request
```

Comment:
Interesting.  When we have params like this I expect the opening brace on the same line.

```new class (
  payload: $this->payload,
) extends Request {
  // ...
}```

Is that possible?

Reply:
Nope. It cannot be depend on params, we can just place it on a new line or not.